source/refined-github.ts

```diff
@@ -177,6 +177,7 @@ import './features/batch-mark-files-as-viewed';
 import './features/unwrap-useless-dropdowns';
 import './features/linkify-notification-repository-header';
 import './features/stop-redirecting-in-notification-bar';
+import './features/stop-redirecting-commit-pull-url-to-commit-url';
```

Comment:
Can you rename the feature?

```suggestion
import './features/prevent-pr-commit-link-loss';
```